api/scripts/fix_addresses.py

```diff
@@ -1,56 +1,67 @@
 import os
 import sys
+from typing import Optional, Sequence
 
 sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
 
 import asyncio
 import logging
 
 import asyncclick as click
-from core.store.retriever import Direction
-from core.store.retriever import IntegerFieldFilter
-from core.store.retriever import Order
+from core.store.retriever import FieldFilter, IntegerFieldFilter
 from databases.core import Database
 
 from notd.chain_utils import normalize_address
 from notd.store.retriever import Retriever
 from notd.store.schema import TokenTransfersTable
+from sqlalchemy.sql.expression import func as sqlalchemyfunc
+
+from notd.store.schema_conversions import token_transfer_from_row
+
 
 
 @click.command()
-@click.option('-s', '--start-block-number', 'startBlockNumber', required=True, type=int)
-@click.option('-e', '--end-block-number', 'endBlockNumber', required=True, type=int)
-@click.option('-b', '--batch-size', 'batchSize', required=False, type=int, default=10000)
+@click.option('-s', '--start-block-number', 'startBlockNumber', required=True, type=int, default=12839820)
+@click.option('-e', '--end-block-number', 'endBlockNumber', required=True, type=int, default=12839850)
+@click.option('-b', '--batch-size', 'batchSize', required=False, type=int, default=15)
 async def fix_address(startBlockNumber: int, endBlockNumber: int, batchSize: int):
     database = Database(f'postgresql://{os.environ["DB_USERNAME"]}:{os.environ["DB_PASSWORD"]}@{os.environ["DB_HOST"]}:{os.environ["DB_PORT"]}/{os.environ["DB_NAME"]}')
     retriever = Retriever(database=database)
-    await database.connect()
 
+    async def token(fieldFilters: Optional[Sequence[FieldFilter]] = None):
+        query = TokenTransfersTable.select()
+        query = query.where(sqlalchemyfunc.length(TokenTransfersTable.c.toAddress) != 42)
+        if fieldFilters:
```

Comment:
i would suggest not using fieldFilters here, just use the query.where structures like above it will be more readable to have:
```
query = (
    db.select(...)
    .where(...)
    .where(...)
    .where(...)
```